pkg/auth/zcapld/service.go

```diff
@@ -18,14 +19,17 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/kms"
 	ariesstorage "github.com/hyperledger/aries-framework-go/pkg/storage"
 	"github.com/hyperledger/aries-framework-go/pkg/vdr/fingerprint"
+	"github.com/trustbloc/edge-core/pkg/log"
 	"github.com/trustbloc/edge-core/pkg/zcapld"
 )
 
 const (
-	storeName            = "zcap-capability"
-	invocationTargetType = "urn:edv:vault"
+	storeName   = "zcap-capability"
```

Comment:
heads up: I don't think all our storage impls support dashes in store names ;)

Reply:
@llorllale I change it to underscore